Apps/Src/SendCarCAN.c

```diff
@@ -48,19 +51,27 @@ uint8_t get_SendCarCAN_Q_Space(void) {
 
 /**
  * @brief Wrapper to put new message in the CAN queue
- * @param message the CAN message to put in SendCarCAN Queue
 */
 void SendCarCAN_Put(CANDATA_t message){
-    OS_ERR err;
+    OS_ERR err = OS_ERR_NONE;
     CPU_TS ticks;
-
-    OSMutexPend(&CarCAN_Mtx, 0, OS_OPT_PEND_BLOCKING, &ticks, &err);
-    assertOSError(err);
+    bool success = false;
+    
+    static uint8_t carcan_ctr = 0;
     
-    bool success = SendCarCAN_Q_put(&CANFifo, message);
+    if(carcan_ctr > 3){
```

Comment:
wait, does this not just put every 3d message onto the bus? What is the point of this?

Reply:
Okay, in order to solve Q saturation issues, the solution is not to just skip 2/3 of all the incoming queue pushes. See if you can throttle the input rate on the sending applications, and more finely control what actually gets skipped. IMO, IOstate data is super low priority, and so is most of the motor's stuff. Add delays / filtering there, and see whether you can handle the traffic.

Reply:
We did this soley for the purposes of testing to see if everything works without needing to set up the software the configure the motor controller to give us less messages less frequently. The counters will be removed and addressed in #386

Reply:
Also, might redesign this in this issue ticket #388